packages/block-editor/src/components/inserter/block-patterns-tab.js

```diff
@@ -178,7 +175,16 @@ function BlockPatternsTabs( {
 	const [ showPatternsExplorer, setShowPatternsExplorer ] = useState( false );
 	const categories = usePatternsCategories();
 	const isMobile = useViewportMatch( 'medium', '<' );
-
+	const getBlockPatternsCategoryPanel = useCallback(
```

Comment:
(I left this comment elsewhere but I can't find it :P)

Is there any reason for the useCallback? I think we should just inline the render function personally unless there's a real measurable impact.

Reply:
Sure, I'll inline. Also your comment was in the commit and then I force pushed :)